### PR TITLE
more warnings about s3 vs sqs

### PR DESCRIPTION
**Pull request by corey-thuen-gravwell**

### CODE REVIEW

**Comment by kris-watts-gravwell** [comment]
assigning to Ashley and/or Fritz for review due to my *Exrlint Spurlin*

File: ingesters/s3.md [hunk 1]
```diff
@@ -7,9 +7,17 @@ myst:
...
+
+S3 buckets are also not meant to provide low latency access to data. The S3 API is basically an HTTP API and most of the systems that write to S3 buckets batch up data and write large chunks at a time.  Be forewarned, you may see delays measured in minutes between when an entry says it was created and when Gravwell can actually consume it due to this common S3 usage pattern.
+
+If you have large amounts of S3 data and/or are interested in "tailing" S3 data, you probably should be using [SQS ingester](/ingesters/sqs) or another mechanism like (Amazon Kinesis)[/ingesters/kinesis].
```
Comment (ashnwade) [inline ingesters/s3.md:18]:
```suggestion
If you have large amounts of S3 data and/or are interested in "tailing" S3 data, you probably should be using the [SQS ingester](/ingesters/sqs) or another mechanism like (Amazon Kinesis)[/ingesters/kinesis].
```

**Review by ashnwade — CHANGES_REQUESTED** [review]
We start off calling this "The Amazon S3 ingester," so it makes sense to refer to this as "the SQS ingester"

**Review by ashnwade — APPROVED** [review]
LGTM

**Pull request merged**